### ISSUE TITLE
Name property for list item in 'Change the active workbook target platform' Combo box is not defined properly in Workbook front page 

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#344]
### Steps to Reproduce

1. Open "Xamarin Workbook"
2. Products window is opened.
3. Navigate to the "wpf "icon and select "create workbook".
4. Navigate to 'Change the active workbook target platform' combo box and Expand it
6. Navigate through the list items and Observe Inspect property

### Expected Behavior
Name property should be properly defined for list item(For e.g -'WPF' )
### Actual Behavior
Name property for Combo box is defined as 'Xamarin.Interactive.Client.ViewControllers.WorkbookAppViewController'

Narrator Behavior: Narrator is announcing as 'Xamarin.Interactive.Client.ViewControllers.WorkbookAppViewController, '
NVDA behavior:NVDA is announcing as 'Combo box'

### User impact: 
User who rely on screenreader will not get to know which list items they are traversing in 'Target platform' combo box.

### Screen-Shots: 
![combobox_list_item_inspect](https://user-images.githubusercontent.com/22956130/40722908-36f1620a-643b-11e8-9c43-8b29e32faa5f.png)
![combobox_list_item_narrator](https://user-images.githubusercontent.com/22956130/40722909-371e4ad6-643b-11e8-8df0-728240d243ff.png)
![combobox_list_item_nvda](https://user-images.githubusercontent.com/22956130/40722910-374b38e8-643b-11e8-9fef-cba25091b5b1.png)

### Environment
Windows :Windows 10 Pro Version 1803 (OS Build : 17672.1000)
Xamarin Version : 1.4 (master/b3f92f9)
NVDA Version: 2018.1

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yMAS
Accessibility
Xamarin-Workbooks-Windows
Wipro
MAS4.2.1

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1813
